lib/index.js

```diff
@@ -6,6 +6,7 @@ const PeerList = require('webrtc-signal-http/lib/peer-list')
 const signalRouterCreator = require('webrtc-signal-http')
 const heartbeatRouterCreator = require('webrtc-signal-http-heartbeat')
 const capacityRouterCreator = require('webrtc-signal-http-capacity')
+const recognitionRouterCreator = require('webrtc-signal-http-peer-identification/lib')

```

Comment:
we should change `package.json#main` to avoid the `/lib` here

Reply:
Fixed!